src/test/java/duke/tasks/DeadlineTest.java

```diff
@@ -0,0 +1,24 @@
+package duke.tasks;
+
+import duke.dukeExceptions.DukeException;
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
I think i'd like to see a line between the two import statements, so I can clearly see a line between duke imports and junit imports.